packages/mobile/src/account/SyncBankAccountScreen.tsx

```diff
@@ -0,0 +1,72 @@
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import { StackScreenProps } from '@react-navigation/stack'
+
+import * as React from 'react'
+import { useAsync } from 'react-async-hook'
+import { useTranslation } from 'react-i18next'
+import { ActivityIndicator, View, Text, StyleSheet } from 'react-native'
+import { createFinclusiveBankAccount, exchangePlaidAccessToken } from 'src/in-house-liquidity'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import useSelector from 'src/redux/useSelector'
+import { mtwAddressSelector, walletAddressSelector } from 'src/web3/selectors'
+
+type Props = StackScreenProps<StackParamList, Screens.SyncBankAccountScreen>
+
+const SyncBankAccountScreen = ({ route }: Props) => {
+  const { t } = useTranslation()
+  const accountMTWAddress = useSelector(mtwAddressSelector) || ''
+  const walletAddress = useSelector(walletAddressSelector) || ''
+  const { publicToken } = route.params
+
+  useAsync(async () => {
+    const accessTokenResponse = await exchangePlaidAccessToken({
+      accountMTWAddress,
```

Comment:
Is it possible that we get to this line if user do not have `accountMTWAddress` ?

Reply:
I don't think so since they will need an accountMTWAddress to do persona and start plaid which have to be done before this step